addons/google_account/google_account.py

```diff
@@ -1,63 +1,65 @@
 # -*- coding: utf-8 -*-
 
+from datetime import datetime
+import logging
+import simplejson
+import urllib2
+import werkzeug
+
 import openerp
 from openerp.http import request
-from openerp.osv import osv
-from openerp import SUPERUSER_ID
-from openerp.tools.translate import _
-from datetime import datetime
+from openerp import models, api, _
 from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT
```

Comment:
use fields.datetime.to_string() or fields.datetime.from_string() instead of DEFAULT_SERVER_DATETIME_FORMAT